Frontend/app/problem/display/displayProblem.component.ts

```diff
@@ -113,18 +117,35 @@ export class DisplayProblemComponent implements OnInit, OnDestroy {
   }
 
   returnToClassPage() {
+    this.sendSuccessUpdate();
     this.activatedRoute.params.subscribe(params => {
       let classCode = params['classCode'];
       this.router.navigateByUrl('/class/' + classCode);
     });
   }
 
+  sendSuccessUpdate() {
+    this.problemService.postProblemSuccess(this.classCode, this.problemCode, this.problemIndex, "True")
+      .subscribe(
```

Comment:
@RyanYoung25 I dont think its needed. The subscribe is only used to tell angular to create a listener for this for whenever it returns. If you dont care what happens im assuming its not needed. Ive never done a case like this myself but I think youd be fine to remove it